setup.sh

```diff
@@ -17,11 +17,11 @@ sudo -u www-data mkdir composer
 # Create folder for wikis
 sudo -u www-data mkdir wikis
 
-# create a database user that is allowed to create databases for each wiki
-sudo mysql -u root --password='' -e "
-CREATE USER 'patchdemo'@'localhost' IDENTIFIED BY 'patchdemo';
-GRANT ALL PRIVILEGES ON \`patchdemo\_%\`.* TO 'patchdemo'@'localhost';
-"
+# Create a database user that is allowed to create databases for each wiki,
+# and the central patchdemo database
```

Comment:
Duplicate comment (line 23)

Reply:
not quite - this says the user is allowed to create the central patchdemo database

the subsequent comment is actually creating it

I guess the comma is a bit ambiguous.